packages/app/cypress/e2e/runner/issues/issue-18042.cy.js

```diff
@@ -4,7 +4,7 @@ import { loadSpec } from '../support/spec-loader'
 describe('issue 18042', () => {
   it('Call count is shown even if cy.stub().log(false)', function () {
     loadSpec({
-      fileName: 'issue-18042.cy.js',
+      filePath: 'issues/issue-18042.cy.js',
```

Comment:
The change from `fileName` -> `filePath` for the helper was made to ensure we could navigate to the appropriate url. Prior to this, selecting the fileName from the list accomplished this for us.